### ISSUE TITLE
[Task]: Linter for 30k labels and milestones

### ISSUE THREAD

**Issue by widal001** [HHS/simpler-grants-gov#823]
### Summary

Create an automated GitHub action linter that checks:
- All 10k deliverable issues marked "in progress" have a corresponding milestone
- All of the issues in the milestones associated with a given 30k deliverable also have a label which matches that 30k

### Blocked by

- [x] #854 

### Acceptance criteria

- [ ] When a new issue is created with the `deliverable: 10k ft` label, a corresponding milestone is created and assigned to it
- [ ] At least once per day a linter runs that:
  - [ ] Finds all issues in the sprint board assigned to a given 10k milestone
  - [ ] Isolates issues that have the wrong value for the deliverable column
  - [ ] Updates their deliverable column value accordingly

**Comment by widal001** [comment]
I opened a draft PR for this work, I've also been testing it in practice on my demo project board:

- Example of a [10k issue for which a new milestone was created and assigned](https://github.com/widal001/project-demo/issues/90)

**Comment by widal001** [comment]
We might be pausing this work based on our decision to move away from the use of 10k issues
